azerothcore/azerothcore-wotlk
# fix(scripts/npc) Experience Eliminator only asks pertinent questions

## Changes Proposed:
-  Fixes the Experience Eliminator NPCs to only ask the right question based on your current XP gain flag.

## Issues Addressed:
None

## SOURCE:
Personal finding.

## Tests Performed:

- builds with latest AzerothCore
- tested with Behsten and Slahtz on appropriate faction characters


## How to Test the Changes:

1. Visit Behsten in Stormwind or Slahtz in Orgrimmar
2. Note that the NPC only asks you the right question
3. Stop/start your experience with the NPC
4. Note that the other question shows now

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
